src/model.ts

```diff
@@ -76,10 +86,21 @@ export abstract class Model<T extends Model<T>> {
     return pk.query()
   }
 
+  secondaries(keys: Exact[]) {
```

Comment:
This is a nice utility method, it would also be pretty nice if it could handle an optional `keys` argument similar to `primary()` method.

Reply:
I'm not sure when this would get called, assuming you pass the same keys to the secondary method as the primary there will always be a primary key in the keys array. Would the purpose be to not run the function if the only key in the array is a primary key?